.changeset/nice-lies-sip.md

```diff
@@ -0,0 +1,6 @@
+---
+"@vercel/next": patch
+"api": patch
```

Comment:
Any idea why `api` is changing here? 

I think we need to modify that since `api` is private so its not published to npm and therefore isn't versioned.

https://github.com/vercel/vercel/pull/10474

Reply:
hmm, not sure, I just clicked the usual button from the bot reply and committed it 🤔

Reply:
Can actually see it encoded in the URL params:

```
https://github.com/vercel/vercel/new/fix/i18n-basepath-fallback-false-404?filename=.changeset/lemon-feet-compare.md&value=---%0A%22%40vercel%2Fnext%22%3A%20patch%0A%22api%22%3A%20patch%0A---%0A%0A%5Bnext%5D%20fix%20ENOENT%20on%20%2F404.html%20when%20%60fallback%3A%20false%60%20w%2F%20%60basePath%60%0A```

Reply:
@TooTallNate Any idea why the changesets bot thought that `api` changed?